.github/workflows/ci.yml

```diff
@@ -441,14 +533,35 @@ jobs:
         run: |
           minikube profile list
           minikube status
-
       - name: Setup for upgrade test
         run: |
           gh auth login --with-token < <(echo ${{ secrets.GITHUB_TOKEN }})
           curl -fsS https://packages.fluvio.io/v1/install.sh | bash
 
-      - name: Start sccache server
-        run: sccache --start-server
+      # Download artifacts
```

Comment:
I am concerned this seems to be duplicated from other steps. can it re-use same steps?

Reply:
I don't think there's a way to abstract a group of steps unfortunately. I think this is about as compact as this can be